server/src/service.rs

```diff
@@ -293,4 +314,42 @@ impl Service {
 
         Ok(())
     }
+
+    pub async fn set_locked(
+        &self,
+        locked: bool,
+        objects: &[OwnedObjectPath],
+    ) -> Result<Vec<OwnedObjectPath>, ServiceError> {
```

Comment:
This should return a `(Vec<OwnedObjectPath>, Vec<OwnedObjectPath>)`

The first is for items that have the state to be changed, and the second for items that won't have it state changed.

Reply:
Done.